google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/JobStatistics.java

```diff
@@ -761,6 +774,11 @@ public SearchStats getSearchStats() {
       return searchStats;
     }
 
+    /** Statistics for metadata caching in BigLake tables. */
```

Comment:
nit:  possibly include a link to more information about metadata caching, e.g. https://cloud.google.com/bigquery/docs/biglake-intro or elsewhere?

Reply:
Added to comment of MetadataCacheStats class.
Thanks for the review.